src/extension.ts

```diff
@@ -36,106 +98,175 @@ export function activate(context: vscode.ExtensionContext) {
-      const commentPattern = /\/\/.*\n/g
-      const comments = text.match(commentPattern)
-      const commentRanges =
-        comments?.map(x => {
-          const startPos = text.indexOf(x)
-          const endPos = startPos + x.length
-          return new vscode.Range(
+      if (featureEnabled('highlightInvalidVariables')) {
+        const diagnosticsMissingKeys: vscode.Diagnostic[] = []
+        // Get all comments in the file in order to not underline missing keys in comments
+        const commentPattern = /\/\/.*\n/g
+        const comments = text.match(commentPattern)
+        const commentRanges =
+          comments?.map(x => {
+            const startPos = text.indexOf(x)
+            const endPos = startPos + x.length
+            return new vscode.Range(
+              file.document.positionAt(startPos),
+              file.document.positionAt(endPos)
+            )
+          }) ?? []
+
+        // Underline missing keys in config.ts file
+        for (const key of missingKeysInDotEnv) {
+          const startPos = file.document.getText().indexOf(`'${key}'`)
+          const endPos = startPos + key.length + 2
+          const keyRange = new vscode.Range(
             file.document.positionAt(startPos),
             file.document.positionAt(endPos)
           )
-        }) ?? []
-
-      // Underline missing keys in config.ts file
-      const diagnostics: vscode.Diagnostic[] = []
-      for (const key of missingKeysInDotEnv) {
-        const startPos = file.document.getText().indexOf(`'${key}'`)
-        const endPos = startPos + key.length + 2
-        const keyRange = new vscode.Range(
-          file.document.positionAt(startPos),
-          file.document.positionAt(endPos)
-        )
 
-        const isMissingKeyInTheComment =
-          comments &&
-          commentRanges.some(commentRange => commentRange.contains(keyRange))
-        if (!isMissingKeyInTheComment) {
-          // Underline missing key in config.ts file. Add a message to go to .env.jsonc file and link it
-          const errorMessage = new vscode.Diagnostic(
-            keyRange,
-            `Key '${key}' is missing in .env.jsonc`,
-            vscode.DiagnosticSeverity.Error
-          )
-          errorMessage.relatedInformation = [
-            {
-              location: new vscode.Location(file.document.uri, keyRange),
-              message: 'Missing key in .env',
-            },
-            {
-              location: new vscode.Location(
-                fileUri,
-                new vscode.Range(0, 0, 0, 0)
-              ),
-              message: 'This file is missing the key',
-            },
-          ]
-          errorMessage.code = {
-            value: 'missing-key',
-            target: fileUri,
-          }
-          errorMessage.source = 'configuru'
+          const isMissingKeyInTheComment =
+            comments &&
+            commentRanges.some(commentRange => commentRange.contains(keyRange))
```

Comment:
Is this lint / prettier issue or different VS Code formatting setup? Can you please check. If this happend because the old code wasn't linted, consider this resolved.